pages/account/manage/milestones.js

```diff
@@ -50,7 +50,7 @@ export default function ManageMilestones({ milestones }) {
       <Page>
         <Navigation />
 
-        <Button href="/account/manage/milestone">
```

Comment:
I don't think this change is needed

Reply:
Yeah, it was not intentional, I mistakenly left a white space there after the quotation. how do I fix it?

Reply:
I've edited this and should reflect when i push after the others has been resolved.

Reply:
ok thanks 👍